app/reducers/users.js

```diff
@@ -0,0 +1,92 @@
+export function userregistration(state = {
+  error: false, 
+  registered: false
+}, action) {
+  switch(action.type) {
+    case 'USER_REGISTERED_SUCCESS':
+    return {
+      registered:true,
+      error:false
+    }
+
+    default:
+    return state;
+
+  }
+}
+
+function processPage(users, pages, page)
+{
+  var listOfUsers = [];
+  if(users && _.size(users) > 0)
+  {  
+   users.map(user => {
+      listOfUsers.push(user.id);
+    })
+  }
+ var result =  Object.assign({}, pages, {[page]: {
+     ids: listOfUsers,
+    fetching: false
+  }});
+  return result;
+}
+
+
+export function users(state={
+  usersById: [],
+  userCount: 0,
+  currentPage: 1,
+  pages: []//pages: { 1: {ids:[23,45,45], fetched: true}}
+}, action) {
+  switch(action.type){
+    case 'USERS_FETCHED':
+      const users = processUsers(action.users, state.usersById);
+      const page = processPage(action.users, state.pages, action.page);
+      //var final = Object.assign {{}, state, users, {pages: page}, {userCount: action.count}};
+      return final;
```

Comment:
@aksanoble users.js is not used actually. UserAuth.js is the reducer used. I'll remove this file in the next commit.